packages/shared/src/components/modals/SharedBookmarksModal.tsx

```diff
@@ -49,10 +49,14 @@ export default function SharedBookmarksModal({
     },
   );
 
-  const [isRssUrlCopied, copyRssUrl] = useCopyLink(
+  const [, copyRssUrl] = useCopyLink(
```

Comment:
This seems off?

Reply:
Lint fix because it throws an error that there's an unused variable and this was the suggest fix